app/controllers/pings_controller.rb

```diff
@@ -53,14 +56,37 @@ def ping_params
     params.require(:ping).permit(:time, :store, :user_id)
   end
 
+  def last_ping
+    @ping = User.find(params[:id]).pings.last
+  end
+
+  def correct_user?
+    @requested_ping = Ping.find(params[:id])
+    if @requested_ping.user_id == current_user.id
+    else
+      render json: {
+        message: "You are not authorized to update another user's ping"
+      },
+             status: 401
```

Comment:
is status supposed to be double indented? if yes, ignore me.

Reply:
Its the formatter that likes it that way 🤪